.cirrus.yml

```diff
@@ -21,3 +20,48 @@ test_task:
     - cargo fmt --check
     - cargo clippy -- -D warnings
   before_cache_script: rm -rf $CARGO_HOME/registry/index
+
+functional_task:
+  required_pr_labels: needs-functional-test
```

Comment:
You can manually trigger the build from Cirrus web UI

Reply:
Unresolving this. I think we need to drop the pr_label. For non-PRs it seems all commits trigger a build and since we're using this repo for development branches this would overload the system unnecessarily. The only alternative we have is manual trigger I'm afraid.